index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>My portfolio</title>
+</head>
+<body>
+    <header class="container">
+        <nav class="container-logo">
+            <a href="#">
+                <img src="images/logo.png" alt="logo">
+            </a>
+            <a href="#">
+                <img src="images/Union.png" alt="union" class="union">
+            </a>
```

Comment:
- [ ] Kindly wrap your hamburger button with a `<nav>` tag instead of an  `<a>` tag. This is important for semantic reasons.